### PR TITLE
first pass at implementing execute method, closes #48

### LINKED ISSUE #48: Execute method not supported?
I was trying to execute a `LogUnsubEvent` call and I time out.  Then I looked inside the code and see this:

https://github.com/ExactTarget/Fuel-Node-SOAP/blob/master/lib/fuel-soap.js#L331-L333

Am I out of luck? :)

### PR DESCRIPTION
**Pull request by nlf**
Also added `grunt-cli` to dev dependencies so I wouldn't have to globally install it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "chai": "~1.9.1",
     "grunt": "~0.4.5",
     "grunt-bump": "~0.0.14",
+    "grunt-cli": "^0.1.13",
```
Comment (vernak2539) [inline package.json]: please change this to "~0.1.13"

**Comment by vernak2539** [comment]
Please add test cases

File: lib/fuel-soap.js [hunk 2]
```diff
@@ -328,8 +328,26 @@ FuelSoap.prototype.describe = function (type, callback) {
...
+FuelSoap.prototype.execute = function (type, props, callback) {
 
+  var body = {
+    ExecuteRequestMsg: {
+			'$': {
+				'xmlns': 'http://exacttarget.com/wsdl/partnerAPI'
+			},
+      Requests: {
+        Name: type,
+        Parameters: props
```
Comment (aydrian) [inline lib/fuel-soap.js]: I know this sounds kind of nitpicky, but why not params instead of props? It just seems to make better since.

Comment (gscott) [inline lib/fuel-soap.js]: I believe @nlf used `props` because that's in keeping with the style of the rest of the module.

**Comment by vernak2539** [comment]
@aydrian @nlf @gscott I see nothing wrong with this PR minus the missing test cases. Once they are added, I will publish a new version

**Comment by nlf** [comment]
added a test.

i had a sporadic issue with the last test in `test/specs/function-soapRequest.js` erroring with an ECONNRESET but it was super inconsistent, and happened to me before i added new tests so i didn't try to fix it.

**Comment by vernak2539** [comment]
published as `v1.1.0`

**Pull request merged**